src/logger/NetHttpClientLogger_test.go

```diff
@@ -2,21 +2,104 @@ package logger
 
 import (
 	"fmt"
+	"net/url"
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
-func TestNetHttpClientLogger(t *testing.T) {
-	testHelper := NewTestHelper()
+//Testing NET HTTP Logger
+
+func TestLogsGet(t *testing.T) {
 
-	testLogger := newLogger()
+	queue := make([]string, 0)
+	options := Options{
+		queue := queue,
+	}
+	netLogger := NewNetHttpClientLogger(options)
+	helper := GetTestHelper()
 
-	testLogger.SetLogFlag(false)
+	resp, err = netLogger.GET(helper.demoURL)
+
+	assert.True(t, helper.parsable(resp))
+	assert.Equal(t, true, strings.Contains(queue[0], "[\"request_method\",\"GET\"]"))
+	assert.Equal(t, true, strings.Contains(queue[0], "[\"request_url\",\""+helper.demoURL+"\"]"))
+	//Dependding on what the get actually gets this could change the response body
+	assert.Equal(t, true, strings.Contains(queue[0], "[\"response_body\",\""+helper.mockHTML+"\"]"))
```

Comment:
are we correct to assume this will be the response body? How do we know this should be true?